docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -141,6 +141,12 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -XX:PerBytecodeRecompilationCutoff=10000
     -Djdk.attach.allowAttachSelf=true
     -Djdk.nio.maxCachedBufferSize=2000000
+    -XX:+UnlockDiagnosticVMOptions
+    -XX:+UseAESCTRIntrinsics
+
+.. note::
+   We enable ``-XX:+UnlockDiagnosticVMOptions`` and ``-XX:+UseAESCTRIntrinsics`` to use a backport introduced by  https://github.com/openjdk/jdk11u-dev/pull/410.
```

Comment:
I would skip link to PR here. Maybe add link to JDK issue (original or backport).